index.html

```diff
@@ -62,14 +62,20 @@ <h1><img src="asset/Mozilla.svg" style="height: 1.04em; vertical-align: -0.23em;
 
       <div class="col-sm-1"></div>
       <div class="col-sm-10">
+
+        <div>
+          <a class="btn btn-primary" href="javascript:open_all()">Expand All</a>
+          <a class="btn btn-primary" href="javascript:close_all()">Collapse All</a>
```

Comment:
Do we do javascript: links or addEventListener?  I don't mind, but I generally use the latter when I can.

Reply:
I've generally had a preference for the former because it's more open to multiple methods of activation (mouse/keyboard/etc.).  I think assistive technologies have generally learned to work around cases where people only listen to `click`, but maybe not in all cases, and I think it's still bad for keyboard users.